docs/core/README.md

```diff
@@ -1,5 +1,53 @@
-# Core Web Component API
+# Clarity Core API
 
 You can find all the Core Web Component APIs documented within this directory.
+Clarity Web Components are designed to work in any front end framework or no
+framework at all. This is an initial getting started guide and will continue to
+improve over the next few months.
 
-* [Demo Dropdown](cwc-test-dropdown.md)
+* [Icons](cwc-icon.md)
+* [Buttons](cwc-button.md)
```

Comment:
I wonder if we should automate either generation or checking this list as we build more components?

Reply:
I'm hoping doing something like this can be part of the docs revamp :)